src/download.js

```diff
@@ -35,6 +35,9 @@ function download_oc() {
       case '4.1':
         url = 'https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.1/linux/oc.tar.gz';
         break;
+      case '4.2':
+        url = 'https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.2/linux/oc.tar.gz';
```

Comment:
My issue not yours . As the oc.tar.gz file is the same name, fails to download if you have already downloaded 4.1

Reply:
this is handled. changes look good,